src/libs/actions/IOU.js

```diff
@@ -2302,6 +2302,10 @@ function navigateToNextPage(iou, iouType, report, path = '') {
     Navigation.navigate(ROUTES.MONEY_REQUEST_PARTICIPANTS.getRoute(iouType));
 }
 
+function submitReport() {
```

Comment:
looks like we already know what will come as a param for this function. Maybe its worth adding it now? not sure...

Reply:
Linter will throw an error, since the param won't be used. This is already implemented in another PR from Artem.